### DISCUSSION TITLE
Trailin Slash Usage in the MindConnect NodJS

### DISCUSSION THREAD

**Discussion by tntstn** [mindsphere/node-red-contrib-mindconnect#182] [General]
I have a question and need an answer immediately. There is a restriction which is defined in insightshub (Insights Hub Release Notes mention change in AssetManager api behaviour) release document. In this document, basicly it says "If there is any usage at the end of the requests with backslash **/** , it should be removed ". As you see below, there is no backslash at the end of the request explained under well-formed requests.

Currently, we are using "@mindconnect/mindconnect-nodejs": "3.22.0". I need to know, is there any usage in this version or if there is an usage in any version, which version is the proper to upgrade?

**Examples of malformed requests:**
GET https://gateway.\Unknown macro: {region}-{environment}.{domain}/api/.../v3/.../{Id}/{Name}**/**

**Examples of well-formed requests:**
GET https://gateway.\Unknown macro: {region}-{environment}.{domain}/api/.../v3/.../{Id}/{Name}

**Comment by sn0wcat** [comment]
The question was asked multiple times. The answer is here: 

https://github.com/mindsphere/mindconnect-nodejs/discussions/355
